packages/components/src/components/Image/images.ts

```diff
@@ -59,6 +59,7 @@ export const PNG_IMAGES = {
     WALLET_2x: 'Wallet@2x.png',
     TOR_ENABLING: 'TorEnabling.png',
     TOR_ENABLING_2x: 'TorEnabling@2x.png',
+    TOR_DISABLE_QUESTION: 'TorDisableQuestion.png',
```

Comment:
Since there were no design I created a quick one, it is probably wrong but at least a placeholder